tests_util/src/db.rs

```diff
@@ -0,0 +1,79 @@
+//! In memory databse which simulates disk database reads
+//! 
+//! According to this article, LevelDB can perform 129.000 random read operations / s
+//! http://www.lmdb.tech/bench/microbench/benchmark.html
+
+use std::{time, thread};
+use std::cell::RefCell;
+use std::collections::{HashMap, HashSet};
+use bloomchain::{Position, Bloom, BloomDatabase};
+use bloomchain::group::{GroupPosition, BloomGroup, BloomGroupDatabase};
+
+/// 1.0s / 129 000
+const LEVELDB_READ_TIME: u32 = 7752;
```

Comment:
Why not make the value explicit instead of precomputing it? A unit in const name would be helpful as well.
```rust
const LEVELDB_READ_TIME_NS: u32 = 1_000_000_000 / 129_000;
```